src/thunderbots/software/ai/primitive/movespin_primitive.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+#include "ai/primitive/primitive.h"
+#include "geom/angle.h"
+#include "geom/point.h"
+
+class MoveSpinPrimitive : public Primitive
+{
+   public:
+    static const std::string PRIMITIVE_NAME;
+    /**
+     * Creates a new MoveSpin Primitive
+     *
+     * @param robot_id The id of the Robot to run this Primitive
+     * @param dest The final destination of the movement
+     * @param final_angle The final orientation the robot should have at the end
```

Comment:
This parameter is `angular_vel`, not `final_angle`. The comment should be updated accordingly.